core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -165,6 +174,7 @@ object ConsumerPerformance {
 
     totalMessagesRead.set(messagesRead)
     totalBytesRead.set(bytesRead)
+    joinTime.set(joinEnd - joinStart)
```

Comment:
Perhaps we should allow for the possibility that we have more than one rebalance?